### ISSUE TITLE
Automerge PR should not be throttled

### ISSUE THREAD

**Issue by ns-ggeorgiev** [renovatebot/renovate#17160]
### What would you like Renovate to be able to do?

When a PR is set to be automerge it should not be blocked based on PRs limits for manual intervention PRs.

### If you have any ideas on how this should be implemented, please tell us here.

This can be achieved in one of the following ways:
1. automerge PRs can be considered with priority over the rest of the PRs. Being automergable, they will likely move faster and unblock the limit for the rest of the PRs
2. automerge PTs can have their own independent limit
3. automerge PRs may simply have no bounds and be created ignoring the limits.

### Is this a feature you are interested in implementing yourself?

Yes

**Comment by rarkins** [comment]
Hi, I would like to consider this to be a duplicate of #10546, which is a more generic solution to the same problem. Can you jump over to that issue to discuss?

**Comment by HonkingGoose** [comment]
Duplicate of #10546
